chapters/6-analysis.tex

```diff
@@ -147,13 +147,19 @@ \subsection{Fitting construction into the data workflow}
 in a unified script, separate from the analysis code, helps
 avoid this and ensures consistency across different outputs.
 
-\subsection{Integrating different data sources}
+\subsection{Combining different data sources}
 
 % When merging is necessary and how to start thinking about it
 To create the analysis dataset,
-it is typically necessary to combine or merge information
-from different data sources.\sidenote{
-	\url{https://dimewiki.worldbank.org/Data_Integration}}
+it is typically necessary to combine information
+from different data sources.
+As discussed in Chapter 3, 
+this process should be documented and 
+communicated about using \textbf{data flowcharts},\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Flow\_Charts}}
+and different data sources should only be combined
+in accordance with the data linkage table.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/Data\_Linkage\_Table}}
 For example, you may merge administrative data with survey data
 in order to include demographic information in your analysis,
 or you may want to integrate geographic information
```

Comment:
the sidenote that starts in the next line should move up (see my earlier comment for a suggestion on where)

Reply:
done in 93480410479541b119828baa56f6dedc28ca3967